new/simple-manager-blueprint.yaml

```diff
@@ -331,6 +347,9 @@ node_templates:
           max_workers: 5
           remote_execution_port: 22
           user: { get_input: agents_user }
+          broker_user: { get_input: rabbitmq_username }
+          broker_pass: { get_input: rabbitmq_password }
+          broker_ssl_cert: { get_input: rabbitmq_cert_public }
 
```

Comment:
you also need to do the same for the openstack blueprint and add the SSL enabled input.